### ISSUE TITLE
Remove community list from explore side bar (spam) 

### ISSUE THREAD

**Issue by CowMuon** [hicommonwealth/commonwealth#3344]
## Problem
The explore sidebar loads in a list of all communities on Commonwealth, in alphabetical order. This has the effect of highlighting pretty much entirely SPAM communities in the sidebar, and it literally loading every single community ever created as part of the initial page load status call, along with every single one of their logos/icons. 

Given that the sidebar will be getting a complete overhaul, let's not ship brokenness. Per Product, remove that community list from sidebar, and do not fetch all that in status route. 

This is literally just ripping this out and tying it off, so setting initial estimation of 1 point. If there are any additional considerations adding to the complexity here, please note in a comment and tag me kudasai. @masvelio @martins87 

Adding React label, but just TBC, we don't need that label anymore, since *everything* is Reactive now.

**Comment by masvelio** [comment]
ripping the list from the side bar is not a big deal. Not sure about removing data from initial call - it might be more complicated as we need to make sure that other places in the app do not use this data. Tbh loading the raw data is not that expensive compared to when you actually open the sidebar. Then hundreds of requests are made and images of all communities are downloaded. 
TLDR: if we want to keep it 1 point: just handle removing the list from the UI. Investigating and messing up with the API call bumps up the points for sure

**Comment by martins87** [comment]
> 

The fetching of all communities is made on explore_sidebar.tsx for the app to display the list of
- starred communities (communities joined and made favorite)
- joined communities (communities joined and *not* made favorite)
- rest of communities (not belonging to two listed above)

This fetching is made every time the user clicks on 'compass' icon, takes less than 1 millisecond (tested a few times) and it's not used in any place other than the sidebar.
